images/manageiq-ui-worker/Dockerfile

```diff
@@ -6,16 +6,22 @@ MAINTAINER ManageIQ https://manageiq.org
 
 ARG NPM_REGISTRY_OVERRIDE
 
+ARG ARCH=amd64
+
 LABEL name="manageiq-ui-worker" \
       summary="ManageIQ user interface worker image"
 
 # Install httpd and remove the existing httpd config from manageiq-appliance
-RUN yum -y install httpd --setopt=tsflags=nodocs && \
+RUN yum -y install httpd sudo --setopt=tsflags=nodocs && \
     rm -f /etc/httpd/conf.d/* && \
     yum clean all
 
 COPY container-assets/npm_registry /tmp/npm_registry
 
+COPY container-assets/phantomjs_centos_7.sh /tmp/phantomjs_centos_7.sh
+
+RUN if [ "${ARCH}" == "ppc64le" ] ; then sh /tmp/phantomjs_centos_7.sh ; fi
```

Comment:
I don't really understand what we're doing here. Can you give more details around why a change like this is needed?

Reply:
there is an internal dependency on phantomjs, binary for which is not present for ppc64le, this is how the build fails for this image if I don't build phantomjs. would it be possible to host this RPM with the PostgreSQL and ruby-install at copr? The phantomjs community itself is no longer active :(

[5/5] Building fresh packages...
error /opt/manageiq/manageiq-ui-service/node_modules/phantomjs-prebuilt: Command failed.
Exit code: 1
Command: node install.js
Arguments:
Directory: /opt/manageiq/manageiq-ui-service/node_modules/phantomjs-prebuilt
Output:
PhantomJS not found on PATH
Unexpected platform or architecture: linux/ppc64
It seems there is no binary available for your platform/architecture
Try to install PhantomJS globally
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
The command '/bin/sh -c source /etc/default/evm &&     cd ${SUI_ROOT} &&     yarn install &&     yarn run available-languages &&     yarn run build &&     yarn cache clean' returned a non-zero code: 1

Reply:
Ah, you're right, there's currently no way to install SUI build dependencies without installing SUI test dependencies.

That said, all you really need is for a `phantomjs` binary to exist in the path, so *if* we don't want phantomjs, I think it can be replaced by `sudo ln -s /bin/true /bin/phantomjs`.

Reply:
@himdel Does this mean that we are currently installing phantomjs as a part of building the SUI in some other way? Why does that way not work for this case? Or is it related to the node install issues?

When I first saw this it looked like we were installing something new.

Reply:
The normal build process does not do the building on the target machine, so we should be fine there. (On regular appliances, SUI `/var/www/miq/vmdb/public/ui/service` is the ouptut of `yarn run build` in SUI, but does not include SUI sources or `node_modules`.)

Reply:
probably resolved here already, but heads up,
dropping phantomjs from service ui in https://github.com/ManageIQ/manageiq-ui-service/pull/1619

Do let me know if backporting it is desired :)